runtime/autoload/health/provider.vim

```diff
@@ -502,7 +502,14 @@ function! s:check_node() abort
           \ ['Install Node.js and verify that `node` and `npm` commands work.'])
     return
   endif
-  call health#report_info('Node: '. s:system('node -v'))
+  let node_v = get(split(s:system('node -v'), "\n"), 0, '')
+  call health#report_info('Node: '. node_v)
+  if !s:shell_error && s:version_cmp(node_v[1:], '6.0.0') < 0
+    call health#report_warn('nodejs host does not support '.node_v)
```

Comment:
Users are going to be confused, most don't know that "nodejs host" is something different than actual "nodejs". It's a bit of mess, but for user-facing messages we should say "Neovim node-client" (the literal repo name) or "Neovim nodejs host".

This is why `:help dev-api-client` now says very clearly:

```
Package Naming                                                                                                                                                  
API client packages should NOT be named something ambiguous like "neovim" or                                                                                    
"python-client".  Use "nvim" as a prefix/suffix to some other identifier                                                                                        
following ecosystem conventions. 
```